python/docs/curves.rst

```diff
@@ -26,12 +26,12 @@ restrictions on the number of channels per frame, or the number of frames in a
 file.
 
 Frames are described by :class:`Frame`-objects. These contain a list of
-:class:`Channel`-objects, which describes the individual curves in more detail.
-Additionally, the :class:`Frame` holds information about the index in which
-*all* the channels in the frame are index against. See :class:`Frame` for a
+:class:`Channel`-objects, which describe the individual curves in more detail.
+Additionally, the :class:`Frame` holds information about the common index such
```

Comment:
Does index contain min/max values, spacing and direction?

I was under impression that by index we mean just index.

Reply:
frame has attributes such as index_min, index_max, spacing and direction. All referring to the index channel

Reply:
From the sentence it sounds like all these values are part of the index